.eslintrc.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   },
   extends: [
     'plugin:vue/essential',
-    '@vue/airbnb',
+    // '@vue/airbnb',
```

Comment:
please don't do this. Checking source code for correctness and readability is a **must** when working with others. When the CI is setup, linting will be checked there. If there is a real reason to exclude the AirBnB linting rules, which have become the defacto standard linting rules, please discuss this with everyone. If you are having trouble with a particular rule, please reach out so we can find a solution.

Reply:
the major issue was when I do ctrl + s to beautify my code everything single quote would be double quotes, so I will let everyone know whenever I do the changes like this.

Reply:
you need to make sure your beautifier respects the linter. Or you could turn it off. We can talk about how to do that.